src/IISIntegration/test/WebSites/shared/SharedStartup/Startup.shared.cs

```diff
@@ -27,7 +29,8 @@ private async Task HostingEnvironment(HttpContext ctx)
 
             await ctx.Response.WriteAsync("ContentRootPath " + hostingEnv.ContentRootPath + Environment.NewLine);
             await ctx.Response.WriteAsync("WebRootPath " + hostingEnv.WebRootPath + Environment.NewLine);
-            await ctx.Response.WriteAsync("CurrentDirectory " + Environment.CurrentDirectory);
+            await ctx.Response.WriteAsync("CurrentDirectory " + Environment.CurrentDirectory + Environment.NewLine);
+            await ctx.Response.WriteAsync("BaseDirectory " + AppContext.BaseDirectory);
```

Comment:
What was AppContext.BaseDirectory before this change?

Reply:
This change didn't affect `AppContext.BaseDirectory`, in both tests it's equal to content root path

Reply:
So for customers on 2.2, can we tell them to use AppContext.BaseDirectory as a work around to having CurrentDirectory.GetCurrentDirectory()?

Reply:
It would work 99% of the time. The only time BaseDirectory would be wrong is when the app is published into subdirectory.

Reply:
subdirectory of what?

Reply:
Like when the app root is not next to the web.config?

Reply:
The app root is required to be next to the web.config, that's the only supported scenario. No?

Reply:
I haven't tried it but I think you could change the path to the app in the web.config. That way you could move the app to a subdirectory.

Reply:
We'd tried that at some point but the web.config's location gave us a lot of trouble. That may have been the publishing tools confusing which directory they should point IIS at.

Also, in-proc is a lot more restrictive on what parameters you can pass in.

Reply:
> The app root is required to be next to the web.config, that's the only supported scenario. No?
 
No, we support apps insider subdirectories just fine.